frontend_lib/src/component/PopinFixed/PopinFixed.styl

```diff
@@ -284,6 +285,23 @@ responsiveColumnStyle()
     & span
       display none
   .wsContentGeneric
+    &__header:has(input)
+      display list-item
+      height fit-content
```

Comment:
I'm a bit confused by the display list-item, what is the reason?

Also, I have tested it on branch and it seems to break the design on xs width

Reply:
Yes, it does break the design for the small screen, by doing so it became possible to change a title on mobile devices, if we wish to keep the design and make it possible to see enough of the entry bar to write a text we will have to reduce the size of the icon witch might then become hard to click on small devices.

The display list-item allow icons to be placed under the title and to still be usable.